### PR TITLE
Update chat-response-quality questions to chat leadership board

### PR DESCRIPTION
**Pull request by abeatrix**
- Migrate test questions from `chat-response-quality` to the leadership board.
- Updated leadership board UI to display chat-questions result.
- Add sonnet 3, gemini pro and gemini flash to chat fixture

image

image

### CODE REVIEW

File: input/chat/question_003/question.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+question: What code do you have access to?
```
Comment (jtibshirani) [inline input/chat/question_003/question.yaml:1]: Could we use the README like we do in `chat-response-quality.test`? That example was taken directly from a user complaint. In general, it'd be good to match the tests as closely as possible because they were carefully designed to trigger certain edge case behaviors.

Also for these evals it seems really nice to use realistic files (all the ones in the original test were taken from Zoekt, for example).

File: input/chat/question_004/question.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+question: how do slices work in go?
+files:
+  - squirrel.ts
```
Comment (jtibshirani) [inline input/chat/question_004/question.yaml:3]: Let's try to match the test here more precisely too and include similar go files!

File: input/chat/question_005/question.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+question: what does this regex do? \n/a|e|i|o|u/\n
+files:
```
Comment (jtibshirani) [inline input/chat/question_005/question.yaml:2]: Same comment here.

**Review by jtibshirani — COMMENTED** [review]
Looks great! Just left a few comments about making the tests match the unit tests as closely as possible.

File: chat-bench.json [hunk 4]
```diff
@@ -1,25 +1,39 @@
...
-  "workspaces": [
-    {
-      "workspace": "input/chat"
-    }
-  ]
+	"queriesDirectory": "queries",
+	"snapshotDirectory": "output/chat",
+	"fixtures": [
+		{
+			"name": "gpt-4o",
```
[the remark this answers is not in the document]
Comment (abeatrix) [reply chat-bench.json:6]: adding!

[the remark this answers is not in the document]
Comment (abeatrix) [reply chat-bench.json:6]: added 
image

**Review by jtibshirani — APPROVED** [review]
